docs/appkit/upgrade/to-reown-appkit-web.mdx

```diff
@@ -351,28 +370,47 @@ function HomePage() {
   return <button onClick={open}>Connect</button>
 }
 ```
+**You can also use a web component to trigger the modal.** You can simply use `<appkit-button />`.
 
-Learn more about Reown AppKit [here](../react/core/installation.mdx)
+Learn more about Reown AppKit [here](../react/core/installation.mdx).
 
 </PlatformTabItem>
 <PlatformTabItem value="vue">
 
-Use your own button with to open the modal
+Use your own button with to open the modal.
 
 ```js
-document.getElementById('my-button').addEventListener('click', () => {
-    modal.open()
-})
-
-<button id="my-button">Connect Wallet</button>
+<template>
```

Comment:
@rtomas, this code snippet seems to be malfunctioned. Can you take a look again?

Reply:
Ignore this comment, sorry!